### ISSUE TITLE
Silly tagging

### ISSUE THREAD

**Issue by cereda** [cereda/sillypage#15]
According to https://github.com/latex3/tagging-project/commit/779fa1f4dfaa782033d3640a2e598160b1be4fd5, `sillypage` is partially compatible with the tagging project (images are missing their corresponding alt texts). Maybe we could work on this? :wink:

**Comment by u-fischer** [comment]
I think it shouldn't be tagged as Figure, but as Span with an ActualText representing the number.

**Comment by cereda** [comment]
Version 1.7 released. Thanks, friends!
